sdk/resourcemanager/Azure.ResourceManager.Core/src/Generated/Models/TagsResource.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Azure.ResourceManager.Core
+{
+    /// <summary>
+    /// A class representing a TagsResource along with the instance operations that can be performed on it.
+    /// </summary>
+    public class TagsResource : TagsOperations
```

Comment:
naming convention is singular noun for a resource then
[Resource].cs
[Resource]Operations.cs
[Resource]Container.cs
[Resource]Data.cs

Reply:
Done